### DISCUSSION TITLE
파이큐티

### DISCUSSION THREAD

**Discussion by kikiru328** [kikiru328/Bone_Detection#57] [Documents / Use case]
https://sites.google.com/site/hyunguk1986/3-sofeware/5-python/-pyqt5-qt-designer

**Comment by hye-jj** [comment]
## 진행상황 2021.11.08
![image](https://user-images.githubusercontent.com/86215536/140666917-488f8095-ca48-4c1c-b2a5-8fba14abb233.png)


- 코드
https://github.com/kikiru328/Bone_Detection/tree/main/%ED%98%9C%EC%A0%95/pyqt_ui

**Comment by hye-jj** [comment]
### pyqt 리소스 파일(qrc) 적용방법 (Windows)
- 이미지 적용 쉬운 방법!!!
https://m.blog.naver.com/PostView.naver?isHttpsRedirect=true&blogId=browniz1004&logNo=221373310932
